common/src/main/java/com/microsoft/identity/common/internal/controllers/TokenOperation.java

```diff
@@ -31,7 +31,7 @@
 import java.util.concurrent.ExecutionException;
 
 public interface TokenOperation {
-    AcquireTokenResult execute() throws InterruptedException, ExecutionException, IOException, BaseException;
+    AcquireTokenResult execute() throws InterruptedException, ExecutionException, IOException, BaseException, Exception;
```

Comment:
I'm not sure how useful it is to throw `Exception` _and_ various subclasses thereof --  are these used in telemetry/logging? Could be we get away with just `Exception`?